apps/passport-client/pages/index.tsx

```diff
@@ -75,21 +75,40 @@ class App extends React.Component<object, AppState> {
   lastBackgroundPoll = 0;
   activePollTimout: NodeJS.Timeout | undefined = undefined;
 
+  // If the extension is installed, set up the devTools object
+  devTools = (window as any).__REDUX_DEVTOOLS_EXTENSION__
```

Comment:
One thing I don't check here is whether this is a dev build. I don't think there's any serious negative implication to having this be present in production builds, especially since regular developer tools also work there, but I'm flagging it just in case anyone else has a strong opinion.

Reply:
I don't have a strong enough opinion to block anything, but I think filtering by build type would be good.  I assume there's probably some code bloat associated with the dependency here, and there could also be bugs/interactions introduced, so it would be best to limit the impact.

Reply:
In this case, there really isn't much overhead. The calls are sending messages to a browser extension, similar to `console.log`, and all of the interesting stuff happens in the browser extension, which is not part of our application bundle.